src/components/animations/ScrollAnimations.tsx

```
@@ -166,13 +184,140 @@ export default function ScrollAnimations({
+      if (timeline && timeline.scrollTrigger) {
+        timeline.scrollTrigger.kill();
+      }
+      if (timeline) {
+        timeline.kill();
+      }
+    });
+    doorTimelinesRef.current = [];
+
+    // 모바일이 아닐 때만 문 애니메이션 생성
+    if (!isMobile) {
+      const sections = container.querySelectorAll('section');
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)